pkg/engine/armvariables.go

```diff
@@ -78,70 +78,17 @@ func getK8sMasterVars(cs *api.ContainerService) (map[string]interface{}, error)
 	isCustomVnet := cs.Properties.AreAgentProfilesCustomVNET()
 	hasAgentPool := len(profiles) > 0
 	hasCosmosEtcd := masterProfile != nil && masterProfile.HasCosmosEtcd()
-	isIPv6DualStackFeatureEnabled := cs.Properties.FeatureFlags.IsFeatureEnabled("EnableIPv6DualStack")
-
-	kubernetesVersion := orchProfile.OrchestratorVersion
-	if cs.Properties.IsAzureStackCloud() {
-		kubernetesVersion = orchProfile.OrchestratorVersion + AzureStackSuffix
-	}
-
-	provisionScriptParametersCommonString := "[concat(" +
-		"'ADMINUSER=',parameters('linuxAdminUsername')," +
```

Comment:
Should we audit all params under `parts/k8s/kubernetesparams.t` & `pkg/engine/params_k8s` (& maybe `pkg/engine/params.go`) and remove all ARM params handling for params that are no longer being used?

Reply:
I did that and removed 2 of them, the rest were still being used one way or another. I think we can reduce further with more refactoring but I stopped there for the purpose of this PR.